app/models/exam_template.rb

```diff
@@ -100,6 +100,59 @@ def split_pdf(path, original_filename=nil, current_user=nil)
     SplitPDFJob.perform_later(self, path, original_filename, current_user)
   end
 
+  def fix_error(filename, exam_num, page_num)
+    error_file = File.join(
+      base_path, 'error', filename
+    )
+    if File.exists? error_file
```

Comment:
This type of error checking is best done with early returns. E.g., `return unless File.exists? error_file`. This prevents the extreme nesting that we currently have here.